static/js/seta.js

```diff
@@ -111,128 +100,107 @@ $(function() {
 
   function buildOSJobMap(joblist) {
     var map = {};
-
-    for (var i = 0; i < joblist.length; i++) {
-      var job = joblist[i];
-      key = fixPlatform(job[0]) + " " + job[1];
-      if (map[key] === undefined) {
-          map[key] = [];
-      }
-      map[key].push(job[2]);
-    }
+    // use groupBy to sort joblist by platform and map them into object
+    map = _.mapObject(_.groupBy(joblist, function(job){
```

Comment:
nit: add a space between ){.

can you briefly comment on what job[0], job[1] and the magic 2 used for _.pluck?